contracts/mechs/nevermined/BalanceTrackerNvmSubscription.sol

```diff
@@ -190,22 +186,21 @@ contract BalanceTrackerNvmSubscription {
 
         // This must never happen
```

Comment:
Why will this never happen?

Reply:
The subscription is issued to the marketplace only, and it's only the MM that processes credits, thus miscalculation must never happen there.